packages/core/admin/admin/src/pages/Webhooks/ListView/index.js

```diff
@@ -276,48 +247,183 @@ function ListView() {
-            <List
-              {...listProps}
-              customRowComponent={props => {
-                return (
-                  <ListRow
-                    {...props}
-                    canUpdate={canUpdate}
-                    canDelete={canDelete}
-                    onCheckChange={handleChange}
-                    onEditClick={handleGoTo}
-                    onDeleteCLick={handleDeleteClick}
-                    onEnabledChange={handleEnabledChange}
-                    itemsToDelete={webhooksToDelete}
-                  />
-                );
-              }}
-            />
+            <Table
+              colCount={5}
+              rowCount={webhooks.length}
+              footer={(
+                <TFooter onClick={() => (canCreate ? handleGoTo('create') : {})} icon={<AddIcon />}>
+                  Add another field to this collection type
+                </TFooter>
+              )}
+            >
+              <Thead>
+                <Tr>
+                  <Th>
+                    <BaseCheckbox
+                      aria-label="Select all entries"
+                      indeterminate={
+                        webhooksToDelete.length > 0 && webhooksToDelete.length < webhooks.length
+                      }
+                      value={webhooksToDelete.length === webhooks.length}
+                      onValueChange={value => handleCheckboxChange('main', value)}
+                    />
+                  </Th>
+                  <Th>
+                    <TableLabel>name</TableLabel>
+                  </Th>
+                  <Th>
+                    <TableLabel>url</TableLabel>
+                  </Th>
+                  <Th width="30%">
+                    <TableLabel>status</TableLabel>
+                  </Th>
+                  <Th>
+                    <VisuallyHidden>actions</VisuallyHidden>
+                  </Th>
+                </Tr>
+              </Thead>
+              <Tbody>
+                {webhooks.map(webhook => (
+                  <Tr key={webhook.id}>
+                    <Td>
+                      <BaseCheckbox
+                        aria-label={`Select ${webhook.name}`}
+                        value={webhooksToDelete?.includes(webhook.id)}
+                        onValueChange={value =>
+                          handleCheckboxChange('secondary', value, webhook.id)}
+                        id={`Select ${webhook.id}`}
+                        name={`Select ${webhook.name}`}
+                      />
+                    </Td>
+                    <Td>
+                      <Text highlighted textColor="neutral800">
+                        {webhook.name}
+                      </Text>
+                    </Td>
+                    <Td>
+                      <Text textColor="neutral800">{webhook.url}</Text>
+                    </Td>
+                    <Td>
+                      <Row>
+                        <Switch
+                          onLabel="Enabled"
```

Comment:
I won't comment about it anymore about it, but every string occurence of a word, in plain english, existing in this file, should go in a translation file 😊